gapis/perfetto/android/trace.go

```diff
@@ -59,8 +61,21 @@ func hasRenderStageEnabled(perfettoConfig *perfetto_pb.TraceConfig) bool {
 	return false
 }
 
-func setupRenderStagesEnvironment(ctx context.Context, d adb.Device, packageName string, perfettoConfig *perfetto_pb.TraceConfig) (app.Cleanup, error) {
+func setupVulkanLayers(ctx context.Context, d adb.Device, packageName string, abi *device.ABI, layers []string) (app.Cleanup, error) {
```

Comment:
I'm a bit confused by this function. Reading it first, I expected it to handle both cases - that is, be the one and only thing that will call `android.SetupLayers`.

However, I don't want to hold up the review due to this, as it does do the right thing.